app/auth_blueprint/models.py

```diff
@@ -26,4 +26,24 @@ def __repr__(self):
         return "<User: {}>".format(self.username) #object instance of the model whenever it is queried
 
 
+class TokenBlackList(db.Model):
```

Comment:
Nice implementation of blacklisting tokens when user logout, although why do you need a column of `date_created` when black listing?

Reply:
Not necessary but keeping track of the logout time